src/games/strategy/triplea/ui/TripleaMenu.java

```diff
@@ -437,6 +438,15 @@ private void addShowCommentLog(final JMenu parentMenu) {
     showCommentLog.setModel(frame.getShowCommentLogButtonModel());
     parentMenu.add(showCommentLog).setMnemonic(KeyEvent.VK_L);
   }
+  
+  /**
+   * @param parentMenu
```

Comment:
nitpick: empty javadoc does not add any value, just a maintenance cost. please remove or alternatively pls add a meaningful description of hte parameter.